implementations/rust/daemon/src/cli.rs

```diff
@@ -181,6 +188,36 @@ impl Args {
     pub fn identity_name(&self) -> String {
         self.identity_name.clone()
     }
+
+    pub fn addon(&self) -> Option<Addon> {
+        self.addon.clone()
+    }
+}
+
+#[derive(Debug, Clone)]
+pub enum Addon {
```

Comment:
Maybe some comments here about what an Addon is?

Reply:
Good call. Lots of places need more comments in `ockamd` and elsewhere. But we'll address all of those after this mile is over.